internal/support/notifications/router.go

```diff
@@ -19,11 +19,13 @@ import (
 	"net/http"
 
 	"github.com/edgexfoundry/edgex-go/internal/pkg"
-	bootstrapContainer "github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/container"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/container"
```

Comment:
This is for DB correct? Should it be renamed to something like /pkg/database/container?
and aliased as dbContainer for consistency...